js/okex.js

```diff
@@ -3544,6 +3546,47 @@ module.exports = class okex extends Exchange {
         };
     }
 
+    async fetchBorrowRates (params = {}) {
+        await this.loadMarkets ();
+        const response = await this.privateGetAccountInterestRate (params);
+        const timestamp = this.milliseconds ();
+        const data = this.safeValue (response, 'data');
+        const rates = [];
+        for (let i = 0; i < data.length; i++) {
+            const rate = data[i];
+            const currency = this.safeCurrencyCode (this.safeString (rate, 'ccy'));
+            rates[currency] = {
+                'currency': currency,
+                'rate': this.safeNumber (rate, 'interestRate'),
+                'span': 86400000,
+                'timestamp': timestamp,
+                'datetime': this.iso8601 (timestamp),
+                'info': rate,
+            };
+        }
+        return rates;
+    }
+
+    async fetchBorrowRate (code, params = {}) {
+        await this.loadMarkets ();
+        const currencyId = this.currency (code)['id'];
+        const request = {
+            'ccy': currencyId,
+        };
+        const response = await this.privateGetAccountInterestRate (this.extend (request, params));
+        const timestamp = this.milliseconds ();
+        const data = this.safeValue (response, 'data');
+        const rate = data[0];
+        return {
+            'currency': currencyId,
+            'rate': this.safeNumber (rate, 'interestRate'),
+            'span': 86400000,
```

Comment:
maybe we should rename span to period, seems more descriptive

Reply:
I would agree with @frosty00